### ISSUE TITLE
Goal not updated when switching sprints on sprints page

### ISSUE THREAD

**Issue by jerone** [keesschollaart81/vsts-sprint-goal#198]
When I'm on the sprint page and I switch to the previous sprint, the sprint goal is not updated.

# Steps to reproduce

1.  Go to "Sprints" page within "Boards".
2.  It should show the current sprint and show the current sprint goal.
3.  Choose another sprint from the sprint dropdown.
4.  See that the sprint goal is not changed.

# Version

Azure DevOps

Sprint Goal (this extension): 5.2.3 (but I could reproduce on the previous version too).

**Comment by keesschollaart81** [comment]
Is it the known bug described at the marketplace page? https://marketplace.visualstudio.com/items?itemName=keesschollaart.sprint-goal

**Comment by jerone** [comment]
Yes, you're right. I'll close this then, if there is another tracking issue?

**Comment by jdshkolnik** [comment]
Is this also the reason the tab label shows sprint goal even when it's off?

![image](https://user-images.githubusercontent.com/2047520/123318405-29eacc00-d4fd-11eb-8519-79be86b758de.png)
